.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "prettier"
+    ],
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
```

Comment:
How are we importing are js code? We're using [module syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) as its the new default and standard